backend/dal/Services/Concrete/BuildingService.cs

```diff
@@ -285,7 +285,6 @@ public Building Update(Building building)
                     {
                         this.ThrowIfNotAllowedToUpdate(parcelBuilding.Building, _options.Project);
 
-                        var parcelBuildings = this.Context.ParcelBuildings.Where(pb => pb.ParcelId == parcelBuilding.ParcelId).ToArray();
```

Comment:
looks like dead code.

Reply:
Just make sure it wasn't being used to lazy-load data for some model

Reply:
Bad way to do it if it was though.